test/test_matmul_cuda.py

```diff
@@ -298,6 +298,165 @@ def _expand_to_batch(t: torch.Tensor):
         # cross comparison
         self.assertEqual(out1_gpu, out2_gpu[0])
 
+
+    @onlyCUDA
+    @skipIfRocm
+    @parametrize("input_dtype", [torch.float32, torch.float16, torch.bfloat16])
+    @parametrize("M", [1, 32, 64])
+    @parametrize("N", [1, 32, 64])
+    @parametrize("K", [1, 32, 64])
+    @parametrize("batch_size", [None, 1, 16])
+    @parametrize("backend", ["cublas", "cublaslt"])
+    def test_mm_bmm_dtype_overload(self, input_dtype, M, N, K, batch_size, backend):
```

Comment:
I didn't touch this code, this is git incorrectly vizualizing the actual changes, which are moving the `grouped_mm` functions out to a new test class.  If needed, can probably figure out how to undo this to get better history.

Reply:
this is annoying, will rework before land